packages/app/src/components/common/table/fields/TimeField.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="info-field-time" :title="utcStringFromISOString(isoString)">
     <span class="time-ago">
-      {{ timeAgo }}
+      {{ isUtcDate ? localDateFromISOString(isoString) : timeAgo }}
```

Comment:
Why do we need it and not simply use `showExactDate` flag?

Reply:
I suggest to drop `showExactDate` and `isUtcDate`, add `format` prop as one of `['time_ago', 'full', 'time_ago_and_full']`. Or `format` as one of `['time_ago', 'full'] and another optional prop `alt_format` as ['time_ago', 'full'].